packages/connectivity/src/scp-cf/cache.ts

```diff
@@ -104,22 +94,11 @@ function isExpired<T>(item: CacheEntry<T>): boolean {
 }
 
 function inferExpirationTime(
-  expirationTime: DateInputObject | undefined
+  expirationTime: number | undefined
 ): number | undefined {
   return expirationTime
-    ? inferExpirationTimeFromDate(expirationTime)
+    ? new Date()
+        .setMilliseconds(new Date().getMilliseconds() + expirationTime)
+        .valueOf()
```

Comment:
[pp] I like to put `const now = new Date();` above. AFAIK `new` does not bind as strongly as calling a function so you probably intend this: `(new Date()).getMilliseconds()` but actually call this: `new (Date().getMilliseconds())`.

Reply:
thank you for your comment. I moved out `new Date()` as another const variable.